content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,211 @@
+    <value>false</value>
+  </property>
+</configuration>
+```
+### Alternative
+I have prepared an example configuration file on [Github](https://github.com/helloezmeral/hpe-binary/tree/main/hadoop-dtap-config). If your Hadoop does not have a special configuration, you can simply download and replace your existing configuration file.
+## Test your HDFS command
+Here are some common commands used to interact with DataTap.
+```bash
+# bash: Current working directory -> $HADOOP_HOME
+
+# Check the version of the hadoop
+bin/hadoop version
+
+# List the files from the default TenantStorage Data Source
+bin/hdfs dfs -ls dtap://TenantStorage/
+
+# Make new directory user in dtap://TenantStorage/
+bin/hdfs dfs -mkdir dtap://TenantStorage/user
+
+# Move the text files helloworld.txt to "cenz" folder
+bin/hdfs dfs -put helloworld.txt dtap://TenantStorage/cenz
+bin/hdfs dfs -put -f helloworld.txt dtap://TenantStorage/cenz # force replacement
+
+# Concatenate a file in dtap
+bin/hdfs dfs -cat dtap://TenantStorage/cenz/helloworld.txt
+
+# Remove a file in dtap
+bin/hdfs dfs -rm dtap://TenantStorage/cenz/helloworld.txt
+```
+
+> Tips:
+> - to get rid of the file path ```bin/```, we can add the Hadoop ```bin``` and ```sbin``` file to ```$PATH```
+> ```
+> export HADOOP_HOME=$HOME/hadoop
+> export PATH=$PATH:$HADOOP_HOME:$HADOOP_HOME/sbin:$HADOOP_HOME/bin
+> ```
+
+> Reference: 
+> [Hadoop File System Shell Document](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/FileSystemShell.html)
+
+# Access DataTaps using pyspark
+## Introduction
+
+## Install ```pyspark```
+There are lots of way to install Spark. The simplest way is to install pyspark package directly using ```pip install pyspark```. Run the following to install the prerequisite packages and pyspark.
+```bash
+# install pyspark & Java
+apt-get install python3 -y
+apt-get install python3-pip -y
+DEBIAN_FRONTEND=noninteractive apt-get install openjdk-11-jdk-headless -y
+pip install pyspark
+```
+There are two ways to interact with pyspark. The first one is to execute the ```pyspark``` command in bash to initiate the pyspark session. The second way is that to treat pyspark as a module which ```python``` kernel can import to. (```import pyspark```)
+
+### Method one: initiate ```pyspark``` session with jars
+In order to use datatap with pyspark, you have to add an external jar file as argument to pyspark. Initiate Spark's interactive shell in python using the following command. 
+```bash
+# bash
+
+# Specify the path of the jars files
+pyspark --jars /opt/bdfs/bluedata-dtap.jar
+```
+After starting the interactive shell, ```Spark Context``` and ```Spark Session``` are automatically initiate for you.
+![image](https://user-images.githubusercontent.com/72959956/120170783-e8d00680-c233-11eb-9fe8-136da9996fdc.png)
+
+We have to specify the Hadoop configurations. After that, you can read files from DataTap just like normally what you did with HDFS.
```

Comment:
don't use the "royal we". Keep consistent as you have been, talking directly to the reader with "you". You might also want to tell the reader why here. For instance... "In order to do such-and-such, you have to specify the Hadoop configurations."

The second sentence needs to be rephrased "After that, you can read files from DataTap just like **you normally** did with HDFS."

The '''py #pyspark... is that supposed to be on this line?

Reply:
Please make the edits that have been recommended here

Reply:
I rewrite it as one sentence

After specifying the Hadoop configurations, you can read files from DataTap just like you normally did with HDFS.

Reply:
About the # pyspark

it is inside the code snippet, and I would like to tell the reader what kind of language am I using, like , bash/pyspark/python etc